overrides/android-override.json

```diff
@@ -35,7 +35,8 @@
                     "funnel_pro_androidrmf_announcement_privacy",
                     "funnel_pro_androidrmf_announcement_vpn",
                     "funnel_pro_androidrmf_announcement_holiday",
-                    "funnel_pro_androidrmf_announcement_wifi"
+                    "funnel_pro_androidrmf_announcement_wifi",
```

Comment:
Once the RMF message is merged, I’ll create another PR to remove the older origins. @cmonfortep